backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -14,6 +14,9 @@ body {
 .admin-nav {
   @include position(absolute, 0 null 0 0);
   width: $width-sidebar;
+  img {
+    max-width: 100%;
+  }
 }
```

Comment:
I'd also use `> img` if possible to avoid the descendant selector for performance reasons.

https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Writing_efficient_CSS#Avoid_the_descendant_selector.21